test/test_quantization.py

```diff
@@ -636,9 +636,13 @@ def checkQuantized(model):
 
 class ObserverTest(QuantizationTestCase):
     @given(qdtype=st.sampled_from((torch.qint8, torch.quint8)),
-           qscheme=st.sampled_from((torch.per_tensor_affine, torch.per_tensor_symmetric)))
-    def test_minmax_observer(self, qdtype, qscheme):
-        myobs = MinMaxObserver(dtype=qdtype, qscheme=qscheme)
+           qscheme=st.sampled_from((torch.per_tensor_affine, torch.per_tensor_symmetric)),
+           reduce_range=st.booleans())
+    def test_minmax_observer(self, qdtype, qscheme, reduce_range):
+        # reduce_range cannot be true for symmetric quantization with uint8
```

Comment:
Just wonder why reduce_range cannot be true for symmetric quantization with uint8? Seems we can use qmin=, qmax=127 with zero_point = 64 to do the quantization.

Reply:
For symmetric quantization, we are constrained to use zero-point = 128 (middle of the 8 bit range). With that value, we cannot eliminate overflow as we will have values greater than 128, which can cause overflow when multiplied with values close to 255.

Reply:
Are we reducing 1-bit of precision from both weights and activations?

Reply:
No, only for the activations. The weight_observer is unchanged and uses all 8 bits